nicsdru_nidirect_theme.theme

```diff
@@ -51,23 +51,35 @@ function nicsdru_nidirect_theme_preprocess_page(array &$variables) {
   // Process search view pages.
   $viewpage = Drupal::routeMatch()->getParameter('view_id');
```

Comment:
I'm surprised this works but perhaps it's because this isn't from a PHP class. The usual and recommended way to reference a service from the global handler is with a `\` prefix... so `\Drupal::service('service.name')`

See https://www.drupal.org/docs/8/api/services-and-dependency-injection/services-and-dependency-injection-in-drupal-8#s-using-global-functions for details